1-mandatory/2-quotegenerator/quotes.js

```diff
@@ -490,3 +478,45 @@ const quotes = [
     author: "Zig Ziglar",
   },
 ];
+
+document.getElementById("newQuoteBtn").addEventListener("click", () => {
+  let randomColor = pickFromArray(coloursArray);
+  let randomQuote = pickFromArray(quotes);
+
+  document.body.style.backgroundColor = randomColor;
+  document.body.style.color = randomColor;
+  document.getElementById("newQuoteBtn").style.backgroundColor = randomColor;
```

Comment:
nicely done on styling the elements in this way, it's something I'm also trying to get the hang of now. really cool to see you do it here.